ppr-ui/src/composables/mhrRegistration/useHomeOwners.ts

```diff
@@ -189,8 +189,8 @@ export function useHomeOwners (isMhrTransfer: boolean = false) {
 
   const hasMinimumGroups = (): boolean => {
     const groups = getTransferOrRegistrationHomeOwnerGroups().filter(group => group.action !== ActionTypes.REMOVED)
-    const hasNoGroups = [HomeTenancyTypes.SOLE, HomeTenancyTypes.JOINT].includes(getHomeTenancyType())
-    return hasNoGroups || !groups || groups.length >= 2
+    return !groups || groups.length >= 2 ||
```

Comment:
I am not sure why we have !groups condition here. If anyone knows?
I just kept it since I was not sure.

Reply:
Not sure why we have !groups, but there are many scenarios for validations, so I assume at some point it was needed.